File: mathenger-api/src/main/java/com/example/mathengerapi/dto/ChatsDTO.java
package com.example.mathengerapi.dto;

import com.example.mathengerapi.models.GroupChat;
import com.example.mathengerapi.models.PrivateChat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ChatsDTO {
    private List<PrivateChat> privateChats;
    private List<GroupChat> groupChats;
}

File: wpf-app/mathenger/Mathenger/Mathenger/DTO/ChatsDTO.cs
using System.Collections.Generic;
using Mathenger.models;
using Newtonsoft.Json;

namespace Mathenger.dto
{
    public class ChatsDTO
    {
        [JsonProperty("privateChats")] public IList<PrivateChat> PrivateChats { get; set; }
        [JsonProperty("groupChats")] public IList<GroupChat> GroupChats { get; set; }
    }
}